cmd/init.go

```diff
@@ -43,6 +44,112 @@ func (h *Handler) initNew(ctx context.Context, req *entity.CommandRequest) error
 	return nil
 }
 
+func (h *Handler) initFromTemplate(ctx context.Context, req *entity.CommandRequest) error {
+	// Prompt for a template to use
+
+	template, err := ui.PromptStarterTemplates()
+	if err != nil {
+		return err
+	}
+
+	// Select GitHub org
+
+	ui.StartSpinner(&ui.SpinnerCfg{Message: "Fetching GitHub scopes"})
+	scopes, err := h.ctrl.GetWritableGithubScopes(ctx)
+	if err != nil {
+		return err
+	}
+	if len(scopes) == 0 {
+		return errors2.NoGitHubScopesFound
+	}
+	ui.StopSpinner("")
+
+	org, err := ui.PromptGitHubScopes(scopes)
+	if err != nil {
+		return err
+	}
+
+	// Enter project name
+
+	name, err := ui.PromptProjectName()
```

Comment:
handle err here (Damn it Golang why don't you warn us about these things!)

Maybe it's time we consider adding a [Linter](https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/errcheck.go)

Reply:
Ugh, yes!